tests/appsec/test_fingerprinting.py

```diff
@@ -77,3 +78,39 @@ def test_session(self):
         assert self.r_create_session.status_code == 200
         assert self.r_user.status_code == 200
         assert all("_dd.appsec.fp.session" in m for m in get_span_meta(self.r_user))
+
+
+@rfc("https://docs.google.com/document/d/1DivOa9XsCggmZVzMI57vyxH2_EBJ0-qqIkRHm_sEvSs/edit#heading=h.32nt1jz5tm2n")
```

Comment:
don't these need to be in a scenario that doesn't have a custom rules file then ?